utility/src/main/java/org/oppia/android/util/accessibility/BUILD.bazel

```diff
@@ -0,0 +1,40 @@
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+load("@rules_jvm_external//:defs.bzl", "artifact")
+
+kt_android_library(
+    name = "accessibility_module",
+    srcs = ["AccessibilityModule.kt"],
+    visibility = ["//visibility:public"],
+    deps = [
+        ":android_accessibility_manager",
+        ":custom_accessibility_manager",
+        artifact("com.google.dagger:dagger:2.27"),
+        artifact("javax.inject:javax.inject:1"),
+    ],
+)
+
+kt_android_library(
+    name = "android_accessibility_manager",
+    srcs = ["AndroidAccessibilityManager.kt"],
+    visibility = ["//visibility:public"],
+    deps = [
+        ":custom_accessibility_manager",
+        artifact("javax.inject:javax.inject:1"),
+    ],
+)
+
+kt_android_library(
+    name = "custom_accessibility_manager",
+    srcs = ["CustomAccessibilityManager.kt"],
+    visibility = ["//visibility:public"],
+)
+
+kt_android_library(
```

Comment:
This should be marked as test only (e.g. with a ``testonly = True`` line) so that it can't be inadvertently included in production builds.

Reply:
Done